### PR TITLE
Getting the food sprite in

### PR DESCRIPTION
**Pull request by Godimas**
Setting up the food sprite. I dont have all the data for it yet, that will take some doing. Basics are in though. Also adjusted the formatting of each sprite to center the title and pretty it up a bit. Also changed Reactors to Nuclear Reactors from a player facing perspective

### CODE REVIEW

File: Other Goodies/Mods/LCDInfo [hunk 1]
```diff
@@ -173,7 +175,14 @@ namespace EconomySurvival.LCDInfo
 
                     var myType = new cargoItemType { item=item, amount=0 };
 
-                    if (type == "Ore")
+                    if (subtypename == "Potato" ^ subtypename.Contains("Meat")) 
+                    {
+                        if (!cargoFoods.ContainsKey(name))
```
Comment (Beersatron) [inline Other Goodies/Mods/LCDInfo:180]: Should this be storing 'subtypename' instead of 'name' ?

File: Other Goodies/Mods/LCDInfo [hunk 2]
```diff
@@ -435,7 +447,7 @@ namespace EconomySurvival.LCDInfo
             var currentDeuterium = deuteriumTanks.Count() == 0 ? 0 : deuteriumTanks.Average(block => block.FilledRatio * 100);
             var totalDeuterium = deuteriumTanks.Count() == 0 ? 0 : deuteriumTanks.Sum(block => block.Capacity);
 
-            WriteTextSprite(ref frame, "Hydrogen Tanks", position, TextAlignment.LEFT);
+            WriteTextSprite(ref frame, "- Hydrogen Tanks -", position, TextAlignment.CENTER);
```
Comment (Beersatron) [inline Other Goodies/Mods/LCDInfo]: hyphen intentional for tanks, instead of square brackets?

Comment (Godimas) [reply Other Goodies/Mods/LCDInfo]: Nope, i effed that up. Same for all the tanks. Fixed.

File: Other Goodies/Mods/LCDInfo (hunk 1 above)
Comment (Godimas) [reply Other Goodies/Mods/LCDInfo:180]: We didnt for the Tools search. If it worked there it should work here too I think, no?

**Pull request merged**